composer.json

```diff
@@ -5,8 +5,8 @@
         "illuminate/validation": "~5.1"
```

Comment:
Now that it's using `Carbon` shouldn't it be required here?

Reply:
I've just noticed that the `composer.lock` file includes Laravel 5.5.2, this will prevent the package from being installed. I think the version constraint here is incorrect and we should probably have `^5.1` (IIRC).

Reply:
It was already being included as a subrequirement of this, but I'll add it separately too